typescript/test/bitcoin.test.ts

```diff
@@ -45,4 +50,145 @@ describe("Bitcoin", () => {
       })
     })
   })
+
+  describe("P2PKH <-> public key hash conversion", () => {
+    const publicKeyHash = "3a38d44d6a0c8d0bb84e0232cc632b7e48c72e0e"
+    const P2WPKHAddress = "bc1q8gudgnt2pjxshwzwqgevccet0eyvwtswt03nuy"
+    const P2PKHAddress = "16JrGhLx5bcBSA34kew9V6Mufa4aXhFe9X"
+
+    const publicKeyHashTestnet = "3a38d44d6a0c8d0bb84e0232cc632b7e48c72e0e"
```

Comment:
No need to use `publicKeyHashTestnet` as you can reuse `publicKeyHash`. The public key hash does not depend on the specific network and will remain the same. Only the address is network-specific.

Reply:
Fixed in c271ec5f1a205a117f7932c1d511bc466658c7fb